filebeat/input/log/log.go

```diff
@@ -111,16 +116,22 @@ func (f *Log) errorChecks(err error) error {
 		return err
 	}
 
+	// At this point we have hit EOF!
```

Comment:
Consider adding this detail to the godoc: `errorChecks determines the cause for EOF errors, and how the EOF event should be handled based on the config options.`